src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -706,16 +706,18 @@ internal override ImmutableHashSet<string> NotNullIfParameterNotNull
         {
             get
             {
-                var attributes = GetAttributes();
+                var attributeInfos = _moduleSymbol.Module.FindTargetAttributes(_handle, AttributeDescription.NotNullIfNotNullAttribute);
```

Comment:
We should instead do what we do for the other nullable attributes: http://sourceroslyn.io/#Microsoft.CodeAnalysis/MetadataReader/PEModule.cs,1186

Reply:
Not sure if the last commit is what you meant.